tests/foreman/ui/test_docker.py

```diff
@@ -338,11 +583,34 @@ def test_publish_once_docker_repo_composite_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        repo_name = gen_string('alpha')
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.organization,
+        ).create()
+        with Session(self.browser) as session:
+            _create_repository(
+                session,
+                org=self.organization.name,
+                name=repo_name,
+                product=self.product.name,
+            )
+            self.assertIsNotNone(self.repository.search(repo_name))
+            self.navigator.go_to_content_views()
+            self.content_views.add_remove_repos(
+                content_view.name, [repo_name], repo_type='docker')
+            self.content_views.publish(content_view.name)
+
```

Comment:
for better readability. Section above refer to non-composite cv and below for composite one